roles/ngi_pipeline/templates/create_ngi_pipeline_dirs.sh.j2

```diff
@@ -17,7 +17,7 @@ fi
 mkdir -p {{ proj_root }}/$1/private/ngi_pipeline/log
 mkdir -p {{ proj_root }}/$1/private/ngi_pipeline/db
```

Comment:
Just wanted to make sure that you had investigated all various possibilities due to seeing those two lines. :)